<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wifi2Status</class>
 <widget class="QDialog" name="Wifi2Status">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WIFI Status</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>399</width>
        <height>930</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="wifi_client_status_group">
         <property name="title">
          <string>Client</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="wifi_client_status_grid">
            <item row="0" column="0">
             <widget class="QLabel" name="wifi_client_status_enabled_label">
              <property name="text">
               <string>Enabled:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="wifi_client_status_ip">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="wifi_client_status_enabled">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="wifi_client_status_status">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="wifi_client_status_subnet_mask">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="wifi_client_status_signal_strength">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="wifi_client_status_signal_strength_label">
              <property name="text">
               <string>Signal Strength:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="wifi_client_status_ip_label">
              <property name="text">
               <string>IP:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="wifi_client_status_subnet_mask_label">
              <property name="text">
               <string>Subnet Mask:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="wifi_client_status_gateway_label">
              <property name="text">
               <string>Gateway:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="wifi_client_status_status_label">
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="wifi_client_status_mac_address_label">
              <property name="text">
               <string>MAC Address:</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="wifi_client_status_rx_label">
              <property name="text">
               <string>RX Count:</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="wifi_client_status_tx_label">
              <property name="text">
               <string>TX Count:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="wifi_client_status_stretch">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="5" column="2">
             <widget class="QLabel" name="wifi_client_status_gateway">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QLabel" name="wifi_client_status_mac_address">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="7" column="2">
             <widget class="QLabel" name="wifi_client_status_rx">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="8" column="2">
             <widget class="QLabel" name="wifi_client_status_tx">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="wifi_ap_status_group">
         <property name="title">
          <string>Access Point</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="wifi_ap_status_grid">
            <item row="7" column="0">
             <widget class="QLabel" name="wifi_ap_status_tx_label">
              <property name="text">
               <string>TX Count:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="wifi_ap_status_enabled_label">
              <property name="text">
               <string>Enabled:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="wifi_ap_status_mac_address_label">
              <property name="text">
               <string>MAC Address:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="wifi_ap_status_gateway_label">
              <property name="text">
               <string>Gateway:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="wifi_ap_status_connected_count_label">
              <property name="text">
               <string>Connected Count:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="wifi_ap_status_ip_label">
              <property name="text">
               <string>IP:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="wifi_ap_status_subnet_mask_label">
              <property name="text">
               <string>Subnet Mask:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="wifi_ap_status_rx_label">
              <property name="text">
               <string>RX Count:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="wifi_ap_status_stretch">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="wifi_ap_status_enabled">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="wifi_ap_status_connected_count">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="wifi_ap_status_ip">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="wifi_ap_status_subnet_mask">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="wifi_ap_status_gateway">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLabel" name="wifi_ap_status_mac_address">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QLabel" name="wifi_ap_status_rx">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="7" column="2">
             <widget class="QLabel" name="wifi_ap_status_tx">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="wifi_mesh_status_group">
         <property name="title">
          <string>Mesh</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QGridLayout" name="wifi_client_status_grid_2">
            <item row="7" column="2">
             <widget class="QLabel" name="wifi_mesh_client_gw">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="15" column="0">
             <widget class="QLabel" name="wifi_mesh_tx_label">
              <property name="text">
               <string>TX Count:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="wifi_mesh_status_label">
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="wifi_mesh_root_node">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLabel" name="wifi_mesh_client_ip">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="wifi_mesh_client_hostname_label">
              <property name="text">
               <string>Client Hostname:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="wifi_mesh_root_node_label">
              <property name="text">
               <string>Root Node:</string>
              </property>
             </widget>
            </item>
            <item row="11" column="2">
             <widget class="QLabel" name="wifi_mesh_ap_sub">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="9" column="2">
             <widget class="QLabel" name="wifi_mesh_ap_ssid">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="wifi_mesh_client_gw_label">
              <property name="text">
               <string>Client Gateway:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="wifi_mesh_status">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QLabel" name="wifi_mesh_client_sub">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="wifi_mesh_client_mac_label">
              <property name="text">
               <string>Client MAC Address:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="wifi_mesh_connected_nodes_label">
              <property name="text">
               <string>Connected Nodes:</string>
              </property>
             </widget>
            </item>
            <item row="13" column="2">
             <widget class="QLabel" name="wifi_mesh_ap_mac">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="wifi_mesh_client_ip_label">
              <property name="text">
               <string>Client IP:</string>
              </property>
             </widget>
            </item>
            <item row="15" column="2">
             <widget class="QLabel" name="wifi_mesh_tx">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="8" column="2">
             <widget class="QLabel" name="wifi_mesh_client_mac">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="wifi_mesh_ap_ip_label">
              <property name="text">
               <string>AP IP:</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="wifi_mesh_ap_ssid_label">
              <property name="text">
               <string>AP SSID:</string>
              </property>
             </widget>
            </item>
            <item row="10" column="2">
             <widget class="QLabel" name="wifi_mesh_ap_ip">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="wifi_mesh_client_sub_label">
              <property name="text">
               <string>Client Subnet Mask:</string>
              </property>
             </widget>
            </item>
            <item row="13" column="0">
             <widget class="QLabel" name="wifi_mesh_ap_mac_label">
              <property name="text">
               <string>AP MAC Address:</string>
              </property>
             </widget>
            </item>
            <item row="12" column="0">
             <widget class="QLabel" name="wifi_mesh_ap_gw_label">
              <property name="text">
               <string>AP Gateway:</string>
              </property>
             </widget>
            </item>
            <item row="11" column="0">
             <widget class="QLabel" name="wifi_mesh_ap_sub_label">
              <property name="text">
               <string>AP Subnet Mask:</string>
              </property>
             </widget>
            </item>
            <item row="12" column="2">
             <widget class="QLabel" name="wifi_mesh_ap_gw">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="14" column="0">
             <widget class="QLabel" name="wifi_mesh_rx_label">
              <property name="text">
               <string>RX Count:</string>
              </property>
             </widget>
            </item>
            <item row="14" column="2">
             <widget class="QLabel" name="wifi_mesh_rx">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="wifi_mesh_connected_nodes">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="wifi_client_status_stretch_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="wifi_mesh_client_hostname">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="wifi_mesh_root_candidate_label">
              <property name="text">
               <string>Root Candidate:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="wifi_mesh_root_candidate">
              <property name="text">
               <string>-</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="wifi_status_button_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>wifi_status_button_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
